crates/oxc_linter/src/snapshots/max_nested_describe.snap

```diff
@@ -0,0 +1,353 @@
+---
+source: crates/oxc_linter/src/tester.rs
+assertion_line: 161
+expression: max_nested_describe
+---
+  ⚠ eslint-plugin-jest(max-nested-describe): Enforces a maximum depth to nested describe calls.
+    ╭─[max_nested_describe.tsx:7:37]
+  6 │                                     describe('quxx', function () {
+  7 │ ╭─▶                                     describe('over limit', function () {
+  8 │ │                                           it('should get something', () => {
+  9 │ │                                               expect(getSomething()).toBe('Something');
+ 10 │ │                                           });
+ 11 │ ╰─▶                                     });
+ 12 │                                     });
+    ╰────
+  help: Too many nested describe calls (6) - maximum allowed is 5
+
+  ⚠ eslint-plugin-jest(max-nested-describe): Enforces a maximum depth to nested describe calls.
+    ╭─[max_nested_describe.tsx:8:41]
+  7 │                                         describe('over limit', function () {
+  8 │ ╭─▶                                         it('should get something', () => {
```

Comment:
Are we intentionally adding diagnostics to `it` and `expect`s as well as `describe`s?

Reply:
In this rule, no. For `expect`, it has another related rule called `max_expect`.

Reply:
In that case, you may want to update your reporting logic to skip diagnostics on `it`, `test`, and `expect`